requirements/docs.txt

```diff
@@ -1,7 +1,5 @@
-# Sphinx 1.7.8 has a bug when building on cached envs
-# https://github.com/sphinx-doc/sphinx/issues/5361
-sphinx>=1.3,!=1.7.8
-numpydoc>=0.6
+sphinx>=2.0
+numpydoc==0.7
```

Comment:
Do we know if these changes are still necessary

Reply:
We don't, but I don't think it's a big deal — docs are mostly for dev work.

Reply:
Happy to approve a different PR with just your last commit, if you want to issue such a thing. =)

Reply:
I'll undo this if it fails. It probably will. I'm mostly worried about the pin on numpydocs